tests/data/mt940/full/cmxl/mt940_1.json

```diff
@@ -35,7 +44,16 @@
         "customer_ref": "NONREF",
         "bank_ref": "55555",
         "supplementary_details": null,
-        "information_to_account_owner": "051?00UEBERWEISUNG?100599?20Gehalt Oktob\ner\n?21Firma\nMustermann\nGmbH?3050060400?31084756\n4700?32MUELLER?34339"
+        "information_to_account_owner": {
+          "transaction_code": "051",
+          "posting_text": "UEBERWEISUNG",
+          "prima_nota": "0599",
+          "purpose": "Gehalt OktoberFirmaMustermannGmbH",
+          "applicant_bin": "50060400",
+          "applicant_iban": "0847564700",
+          "applicant_name": "MUELLER",
+          "return_debit_notes": "339"
+        }
```

Comment:
When I generate these with `sta2json`, I get a bunch of null fields here (which makes sense). How did you generate these files? Do we want to omit null fields?

Reply:
Oops, I added the serde_with import without testing. Turns out ordering is important, I fixed it in a new commit.